### ISSUE TITLE
Scuff-static Cache file not found

### ISSUE THREAD

**Issue by Deadevilgrounds** [HomerReid/scuff-em#138]
I wonder if there is an issue with the cache option in scuff-static.
I have used the cache option fine with scuff-rf but under scuff-static it always complains about "could not open file ..." and after each run no cache file is created in the directory.

I have checked the folder permissions and they are fine and identical to the scuff-rf one which works.

This is the line I am using

`scuff-static --Geometry Trap.scuffgeo --EPFile Trap.ep --PotentialFile TrapGND.potfile --FileBase GND --Cache trap.cache`

**Comment by HomerReid** [comment]
Thanks for the bug report. I'll keep this in mind. I consider it a low-priority issue, as (in my experience) the caching does not actually achieve very significant time savings.

**Comment by HomerReid** [comment]
Actually in looking into this just now I realized that SCUFF-STATIC actually doesn't use caching anyway. (It's mostly useful for full-wave problems in which you want to study the same geometry at multiple frequencies.) The presence of the `--cache` command-line options was an oversight on my part. I've removed it from the code and updated the documentation.

If you are having problems associated with slow BEM matrix assembly for electrostatic problems and you think it might be possible to accelerate, feel free to file a separate issue for that.

Also, a recently-added feature is [implicit support for (infinite-area) multilayered dielectric substrates with optional ground planes](http://homerreid.github.io/scuff-em-documentation/examples/ImplicitSubstrate/ImplicitSubstrate/), so if your structures involve substrates this may speed up the calculations.
